docs/current/guides/128409-build-test-publish-php.md

```diff
@@ -479,7 +479,7 @@ This command:
 
 - initializes a new Dagger Engine session;
 - sets the `DAGGER_SESSION_PORT` and `DAGGER_SESSION_TOKEN` environment variables;
-- executes the PHP pipeline script in that session.
+- executes the PHP pipeline script in that session and displays live progress.
```

Comment:
Do you also want to update the two shell snippets below to match this?

It's OK if you want to defer it for now, but this comment made me check the entire doc and noticed that we are still using the output before `dagger run`.

Reply:
Rather than modifying the output with the Dagger logs (I tried it, it looks noisy), I've added `--silent` to the command.